### DISCUSSION TITLE
Can't update the position of a static body?

### DISCUSSION THREAD

**Discussion by peter1745** [jrouwe/JoltPhysics#471] [Q&A] [answered]
Hello! I have a small question regarding static bodies in Jolt, is it not possible to directly set the position of a static body during runtime? We have a test where we try and do that using `BodyInterface::SetPosition`, it does work for dynamic bodies but doesn't seem to work for static bodies, just wondering if this is intended.

I understand why it wouldn't be supported, since the bodies are supposed to be completely static, and I realize it could cause instability if you accidentally set the position to be inside another body, just wanted to double check that this is actually the case, and that it isn't a problem with our engine code 🙂

**Comment by jrouwe** [comment, marked as answer]
That should just work and as far as I can see it does. Replace these files in the samples app and you'll see that static bodies can move:

[KinematicTest.zip](https://github.com/jrouwe/JoltPhysics/files/10961803/KinematicTest.zip)

It looks a bit weird because they don't transfer velocity to the dynamic bodies, but the dynamic bodies are pushed aside.

**Reply by peter1745** [reply]
Ah yeah you're right, after taking a look at the DebugRendererRecorders output I realize that it was in fact updating the position of the body, but I was simply forgetting to update the position we use for rendering the mesh, but thanks for confirming that it's possible!
